teradata/tests/conftest.py

```diff
@@ -0,0 +1,138 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import csv
+import json
+import os
+import re
+import sys
+import time
+from contextlib import contextmanager
+from copy import deepcopy
+
+import mock
+import pytest
+
+from datadog_checks.dev import get_here
+
+from .common import E2E_METADATA, TERADATA_DD_PW, TERADATA_DD_USER, TERADATA_SERVER, USE_TD_SANDBOX
+
+TABLE_EXTRACTION_PATTERN = re.compile(r'SELECT .* FROM \w+\.(\w+)')
+HERE = get_here()
+
+sys.modules['teradatasql'] = mock.MagicMock()
```

Comment:
let's have a fixture do this with `autouse=True` and lazily import elsewhere if need be

Reply:
I ended up adding `teradatasql` as a dev dependency, since I wasn't able to successfully make a fixture for the mocked module